apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx

```diff
@@ -53,6 +54,21 @@ export const ProposalsList = ({ proposals }: ProposalsListProps) => {
           <Button variant={'primary'}>{t('NewProposal')}</Button>
         </Link>
       </div>
+      <div>
+        <p>
+          {t(
+            `The Vega network is governed by the community. View active proposals, vote on them or propose changes to the network.`
+          )}{' '}
```

Comment:
(non-blocking): do you need this space?
```suggestion
          )}
```